pyproject.toml

```diff
@@ -0,0 +1,62 @@
+[tool.poetry]
+name = "csrank"
+version = "1.2.1"
+description = "Context-sensitive ranking and choice"
+authors = ["Karlson Pfannschmidt <kiudee@mail.upb.de>"]
+license = "apache-2.0"
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "Natural Language :: English",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Software Development",
+]
+repository = "https://github.com/kiudee/cs-ranking"
+documentation = "https://kiudee.github.io/cs-ranking/"
+
+[tool.poetry.dependencies]
+python = "~3.7"
+numpy = "^1.12.1"
+scipy = "^1.5.2"
+scikit-learn = "^0.23.2"
+scikit-optimize = "^0.7.4"
+docopt = "^0.6.2"
+joblib = "^0.16.0"
+tqdm = "^4.11.2"
+keras = "~2.3"
+tensorflow = "^1.5"
```

Comment:
Shouldn't this be <2?

Reply:
`^1.5` is short form for `>=1.5 <2.0.0`. It is using SemVer for dependency specification and a jump for the major version would be compatibility breaking (for more details see [here](https://python-poetry.org/docs/dependency-specification/)).